docs/services/chat/dingding/index.md

```
@@ -0,0 +1,34 @@
+# Dingding
+
+Dingding(i.e. DingTalk China Version) notification service.
+
+The dingding service supports both the "custombot" (自定义机器人) and "worknotice" (工作通知) types of Dingding notifications.
+The service is configured using a URL with the following format:
+
+## Custom Bot (自定义机器人) URL format
+
+!!! info ""
+    dingding://<access_token like abcdef123...>?[kind=custombot][secret=<secret like SECabcdef123...>][&keyword=<keyword>][&title=<title>][&template=<template>]
```

markdownlint-cli2 (0.22.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "&template="

(MD052, reference-links-images)